.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -12,7 +12,8 @@ about: Create a report to help us improve
 
 **Tell us about your environment**	
 * **ESLint version:**	
-* **eslint-plugin-vue version:**	
+* **eslint-plugin-vue version:** 
+* **vue-eslint-parser version:** 
```

Comment:
I like this approach!
But, I think that we should be supported to provide easily with reporting tool like `vue info`.
Maybe we just need to write as the notes.

https://github.com/vuejs/eslint-plugin-vue/commit/5d26e943bcba5544d820e9152a796582cc9a4beb#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R49